docs/self-hosted/config-options.md

```diff
@@ -480,7 +480,7 @@ than you would cache database content. To learn more, see [Assets](#assets).
 | `CACHE_ENABLED`                              | Whether or not data caching is enabled.                                                                                 | `false`                              |
 | `CACHE_TTL`<sup>[1]</sup>                    | How long the data cache is persisted.                                                                                   | `5m`                                 |
 | `CACHE_CONTROL_S_MAXAGE`                     | Whether to not to add the `s-maxage` expiration flag. Set to a number for a custom value.                               | `0`                                  |
-| `CACHE_AUTO_PURGE`<sup>[2]</sup>             | Automatically purge the data cache on `create`, `update`, and `delete` actions.                                         | `false`                              |
+| `CACHE_AUTO_PURGE`<sup>[2]</sup>             | Automatically purge the data cache on `create`/`update`/`delete` actions, Content Version saves, and the sort utility.  | `false`                              |
```

Comment:
I thought about a simpler sentence like this

```suggestion
| `CACHE_AUTO_PURGE`<sup>[2]</sup>             | Automatically purge the data cache on actions that manipulate the data.  | `false`                              |
```

WDYT?

Reply:
```suggestion
| `CACHE_AUTO_PURGE`<sup>[2]</sup>             | Automatically purge the data cache on actions that manipulate the data.                                                 | `false`                              |
```

Sounds good 👍 will update it via this suggestion with formatted spacing.

Reply:
Thankssss 🐍 ❤️